### PR TITLE
Feature/gen 297

### PR DESCRIPTION
**Pull request by dave-at-koor**
All about sales

### CODE REVIEW

File: docs/sales/index.md [hunk 1]
```diff
@@ -6,6 +6,68 @@ title: Sales
...
+On the other end, we can do discrete **one-off services**. We might do a system assessment on an existing cluster and make recommendations. We might troubleshoot a performance issue and suggest a plan to improve. For these types of services, our price will be based on an hourly rate and estimate of the hours it will take.
+
+This piecemeal work might be "off of the menu," so to speak, something we work out when the customer "just needs a little help." We would prefer an annual agreement, but might see this as a step toward a long-term engagement.
+
+We will also do extra projects that go above and beyond. Examples of such projects and higher service levels include:
+
+- 24/7 emergency response SLAs.
+- Data migration where we set up and manage the move.
+- Disaster recovery planning, practice drills, and actual recovery from data loss due to circumstances outside of normal Ceph operation
+  - The normal problems, like disc failures, are handled under **Managed Ceph**. These should not be disasters because we are doing preventative maintenance.
```
Comment (aaron-at-koor) [inline docs/sales/index.md:30]: s/disasters/disasterous/

Comment (aaron-at-koor) [reply docs/sales/index.md:30]: Actually, re-reading, never mind.  It is fine as is.

**Review by aaron-at-koor — APPROVED** [review]
Found one wording error.  Left comment for correction.  And have since decided it was not needed.  Looks good to me.

**Review by zalsader — APPROVED** [review]
Looks good! Ambitious but we can do it 💪

**Comment by IanEff** [comment]
Looks good on first read!  I think there're a few typos, though.  For example:
"The Data Control Center is a materialization of the data control plane, which more of an abstract notion."  The wording is amazing-- it concisely conveys our passion and excitement about the tech we work with-- but it needs an "is..!"
I'll take another pass at it once the network's back up!

**Comment by dave-at-koor** [comment]
> Looks good on first read! I think there're a few typos, though. For example: "The Data Control Center is a materialization of the data control plane, which more of an abstract notion." The wording is amazing-- it concisely conveys our passion and excitement about the tech we work with-- but it needs an "is..!" I'll take another pass at it once the network's back up!

I am not sure what you are referencing (but I can search). To make it easier, you can start a review and make the suggestion inline to show where it belongs. Then submit the review with one of the options. That will also give you the credit for reviewing.

**Comment by dave-at-koor** [comment]
> > Looks good on first read! I think there're a few typos, though. For example: "The Data Control Center is a materialization of the data control plane, which more of an abstract notion." The wording is amazing-- it concisely conveys our passion and excitement about the tech we work with-- but it needs an "is..!" I'll take another pass at it once the network's back up!
> 
> I am not sure what you are referencing (but I can search). To make it easier, you can start a review and make the suggestion inline to show where it belongs. Then submit the review with one of the options. That will also give you the credit for reviewing.

Ah, you were reading the product roadmap, which wasn't changed for this PR. So useful but unrelated.

**Pull request merged**